src/validations/access-checker/wearables.ts

```diff
@@ -74,35 +66,10 @@ type WearableCollectionItem = {
   contentHash: string
 }
 
-const alreadySeen = (resolvedPointers: SupportedAssets[], parsed: SupportedAssets): boolean => {
-  return resolvedPointers.some((alreadyResolved) => resolveSameUrn(alreadyResolved, parsed))
-}
-
-const resolveSameUrn = (alreadyParsed: SupportedAssets, parsed: SupportedAssets): boolean => {
-  const alreadyParsedCopy = Object.assign({}, alreadyParsed)
-  const parsedCopy = Object.assign({}, parsed)
-  alreadyParsedCopy.uri = new URL('urn:same')
-  parsedCopy.uri = new URL('urn:same')
-  return JSON.stringify(alreadyParsedCopy) == JSON.stringify(parsedCopy)
-}
-
-const parseUrnNoFail = async (urn: string): Promise<SupportedAssets | null> => {
```

Comment:
Are we supporting all types of assets now?

Reply:
Nope. I separated parsing/validation. [Here](https://github.com/decentraland/content-validator/pull/42/files#diff-4b211f63323ae0c37eb6d4b2f2865e582de302a81e027af2774755a72221ca4fR438)